bolt-modules/boltlib/lib/puppet/functions/catch_errors.rb

```diff
@@ -8,8 +8,7 @@
 Puppet::Functions.create_function(:catch_errors) do
   # @param error_types An array of error types to catch
   # @param block The block of steps to catch errors on
-  # @return Undef If an error is raised in the block then the error will be
-  # returned, otherwise the result will be returned
+  # @return if an error is raised in the block then the error will be returned, otherwise the result will be returned
```

Comment:
capitalize `if` ?

Reply:
Huh - I thought the other function I looked at had it lower cased but turns out I just looked wrong :woman_shrugging: